### DISCUSSION TITLE
How to clear individual picture thumbnails?

### DISCUSSION THREAD

**Discussion by sugatam** [bpatrik/pigallery2#714] [Q&A] [answered]
Here's my issue. I have a folder - "Foo", that has a whole bunch of photos oriented incorrectly. These were taken by an old camera that didn't set the Orientation flag properly, so all of them have Orientation:1 in their EXIF data. I take one photo and update its exif Orientation to 6, which should be the correct orientation.

In the folder view, ie, when I'm inside "Foo", the thumbnail is messed up - its changing the image dimensions to portrait, but reusing the old pixel data so it looks like a squished version of the original mis-rotated image. If I rename the folder, pigallery2 does the right thing, but no amount of database resetting and re-indexing / temp folder cleaning, etc is picking up the proper image. Clearly it is a caching/db issue, otherwise a folder rename wouldn't have fixed it, but for the life of me I can't figure out how to clear the cache. I really don't want to delete sqlite.db and start over if I can help it.

**Comment by bpatrik** [comment, marked as answer]
You can manually delete the effected folder in your temp folder (the one
that you set for the app)

-- Sorry for being brief, sent from my phone.

**Comment by sugatam** [comment]
Thank you Patrik, that was simple enough. Any way to clear the client-side cache? I tried clearing the Application Storage via (chrome) dev tools but it doesn't seem to help.

**Comment by sugatam** [comment]
Never mind, a disk cache clearing did the trick! Thanks again
